discovery_engine_core/core/src/stack/filters/semantic.rs

```diff
@@ -179,17 +138,18 @@ fn cut_tree(dendrogram: &Dendrogram<f32>, threshold: f32) -> Vec<usize> {
 
 /// Configurations for semantic filtering.
 pub(crate) struct SemanticFilterConfig {
-    /// Cluster cutoff threshold for dissimilarity of cosine distances.
-    distance_threshold: f32,
-    /// Cluster cutoff threshold for dissimilarity of date distances.
-    date_threshold: f32,
+    /// Maximum days threshold after which documents fully decay (must be non-negative).
+    max_days: f32,
+    /// Cluster cutoff threshold for dissimilarity of normalized combined distances (must be in the
+    /// unit intervall [0, 1]).
```

Comment:
```suggestion
    /// unit interval [0, 1]).
```

there is also a small typo in l 114 `initial` that i missed